### PR TITLE
Update NodeJS version for openshift/prometheus master

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

**Comment by coleenquadros** [comment]
/pj-rehearse pull-ci-openshift-prometheus-master-react-app-test

**Comment by simonpasquier** [comment]
/pj-rehearse pull-ci-openshift-prometheus-master-react-app-test

**Comment by coleenquadros** [comment]
/pj-rehearse pull-ci-openshift-prometheus-master-react-app-test

**Comment by simonpasquier** [comment]
/pj-rehearse pull-ci-openshift-prometheus-master-react-app-test
/retest

**Comment by simonpasquier** [comment]
/pj-rehearse pull-ci-openshift-prometheus-master-react-app-test

File: ci-operator/config/openshift/prometheus/openshift-prometheus-master.yaml [hunk 1]
```diff
@@ -49,7 +49,7 @@ tests:
     from: src
 - as: react-app-test
   commands: |
-    nodejs_version=v16.13.1
+    nodejs_version=v20.5.1
```
Comment (rexagod) [inline ci-operator/config/openshift/prometheus/openshift-prometheus-master.yaml:52]: (nit) `v20.14.0` seems to be the latest LTS release: https://nodejs.org/en/download/package-manager, but not a blocker.

File: ci-operator/config/openshift/prometheus/openshift-prometheus-master.yaml [hunk 2]
```diff
@@ -64,6 +64,8 @@ tests:
     export PATH=$PATH:/tmp/node_modules/.bin
     cd -
 
+    echo "NodeJS version: $(node -v)"
```
Comment (rexagod) [inline ci-operator/config/openshift/prometheus/openshift-prometheus-master.yaml:67]:
```suggestion
    echo "NodeJS version: $(node -v)"
    echo "npm version: $(npm -v)"
```

File: ci-operator/config/openshift/prometheus/openshift-prometheus-master.yaml [hunk 3]
```diff
@@ -64,6 +64,8 @@ tests:
...
+
     export npm_config_cache="/tmp/.npm"
```
Comment (rexagod) [inline ci-operator/config/openshift/prometheus/openshift-prometheus-master.yaml:69]:
```suggestion
    export npm_config_cache="/tmp/.npm"
    echo "npm version: $(npm -v)"
```

File: ci-operator/config/openshift/prometheus/openshift-prometheus-master.yaml (hunk 2 above)
Comment (rexagod) [inline ci-operator/config/openshift/prometheus/openshift-prometheus-master.yaml:67]:
```suggestion
    echo "NodeJS version: $(node -v)"
    echo "npm version: $(npm -v)"
```

File: ci-operator/config/openshift/prometheus/openshift-prometheus-master.yaml (hunk 1 above)
Comment (simonpasquier) [reply ci-operator/config/openshift/prometheus/openshift-prometheus-master.yaml:52]: Prometheus upstream installs v20.5.1 which is why I picked it.

**Comment by simonpasquier** [comment]
/pj-rehearse pull-ci-openshift-prometheus-master-react-app-test

**Review by rexagod — APPROVED** [review]
/lgtm

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/52616/rehearse-52616-pull-ci-openshift-prometheus-master-react-app-test/1797539995804766208/artifacts/test/build-log.txt

**Pull request merged**